public/doc/openapi-3-v1.0.0.json

```diff
@@ -4005,6 +4005,32 @@
           }
         }
       },
+      "Tenant": {
```

Comment:
Hmm... I don't see any references to `Tenant` in the rest of the doc.  I wonder if we should have a separate doc or just allow all attributes for the internal API

Reply:
Maybe? The best thing to do would probably be to have a separate openapi spec for the internal API, this is just what we've been doing already in the v0.0 openapi spec

Reply:
Do we even need the `internal` API on Topo now that authentications have moved to Sources?  `tenants` feels like a thing that can be public as long as a logged in user can only see their tenant.

Reply:
There was/is a requirement for internal services (AiOps) to get a list of all tenants so that they can then get all vms.  This definitely isn't the most efficient way of doing this and they're looking into better ways like requesting a dump of entire tables/databases to storage but currently they require this, cc @Ladas are you guys still using this?

Reply:
Yes, this is being used for current usecases. It's not clear yet, what will be the new way.